timeflux_rasr/estimators/blending.py

```diff
@@ -97,13 +97,22 @@ def transform(self, X):
             raise ValueError("X.shape should be (n_trials, n_samples, n_electrodes).")
 
         if Ne != self.n_channels_:
-            raise ValueError('Shape of input is different from what was seen in `fit`')
+            raise ValueError("Shape of input is different from what was seen in `fit`")
 
         if self.window_overlap > 0:  # apply blending only if samples are overlapping
             # estimate the blending coefficients
             # blending is doing an temporal sinus-cosinus interpolation between two redundant samples.
             # if self.window_overlap is 1, it is an average between the last and first sample (blend_coeff is 0.5)
-            blend_coeff = (1 - np.cos(np.pi * (np.arange(1, self.window_overlap + 1) / (self.window_overlap + 1)))) / 2
+            blend_coeff = (
+                1
+                - np.cos(
+                    np.pi
+                    * (
+                        np.arange(1, self.window_overlap + 1)
+                        / (self.window_overlap + 1)
+                    )
+                )
+            ) / 2
```

Comment:
I think that the reformating of this line make the code unreadible.

Reply:
Agree. But maybe this formula should be broken down into manageable parts?